### PR TITLE
LPS-150839 ci test for petra-apache-http-components

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test

**Comment by julschong** [comment]
@kyle-miho
Hi Kyle, 
relevant and default test won't run :(
Need your help on this one!
` [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':apps:document-library:document-library-document-conversion:jar'.
     [exec] > Bundle com.liferay.document.library.document.conversion-4.0.16.jar has errors`

I did change the build.gradle on this module

`       compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations", version: "1.4.0"
	compileOnly group: "org.osgi", name: "osgi.core", version: "6.0.0"
   - -  compileOnly project(":apps:petra:petra-apache-http-components") - -
	compileOnly project(":apps:portal-configuration:portal-configuration-upgrade-api")
	compileOnly project(":apps:static:portal-configuration:portal-configuration-metatype-api")
	compileOnly project(":apps:static:portal:portal-upgrade-api")
	compileOnly project(":core:petra:petra-io")
  + + compileOnly project(":apps:translation:translation-api") + +
`

The module document-library-document-conversion did compile successfully when I ran `gw comopileJava` locally

**Comment by kyle-miho** [comment]
xec] > Task :apps:document-library:document-library-document-conversion:autoUpdateXml SKIPPED
     [exec] error  : Can not find JAR file 'com.liferay.petra.apache.http.components-[0-9.]*.jar'
@julschong i think compileJava may not be enough, since i think the failing gradle task autoUpdateXml caused this, by removing petra-apache-http-components

**Comment by julschong** [comment]
@kyle-miho thanks Kyle, I will see what I missed in that direction

**Comment by julschong** [comment]
changed bnd.bnd -include-resource

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test

**Comment by kyle-miho** [comment]
@julschong can you rebase your master branch and/or resubmit so i can see the unique commits more easily? thanks!

File: modules/apps/document-library/document-library-document-conversion/bnd.bnd [hunk 1]
```diff
@@ -17,7 +17,7 @@ Require-Capability:\
 		filter := "(&(osgi.extender=osgi.configurator)(version>=1.0)(!(version>=2.0)))"
 -includeresource:\
 	OSGI-INF/configurator=configurator,\
-	@com.liferay.petra.apache.http.components-[0-9.]*.jar,\
```
Comment (julschong) [inline modules/apps/document-library/document-library-document-conversion/bnd.bnd:20]: @kyle-miho was this line that caused it

Comment (kyle-miho) [reply modules/apps/document-library/document-library-document-conversion/bnd.bnd:20]: possibly if it was not removed before

**Comment by kyle-miho** [comment]
im going to re-run relevant

**Comment by kyle-miho** [comment]
ci:test:relevant

**Comment by julschong** [comment]
submitted to core https://github.com/liferay-core-infra/liferay-portal/pull/557

File: modules/apps/document-library/document-library-document-conversion/bnd.bnd (hunk 1 above)
Comment (julschong) [reply modules/apps/document-library/document-library-document-conversion/bnd.bnd]: yup thats it :D

**Pull request closed without merging**